src/infrastructure/localSecretsManagerService.ts

```diff
@@ -0,0 +1,16 @@
+import SecretsManagerServiceInterface from '../interfaces/SecretsManagerService';
+
+export default class LocalSecretsManagerService implements SecretsManagerServiceInterface {
+  async getSecret(): Promise<string> {
+    return Promise.resolve(
+      JSON.stringify({
+        username: 'root',
+        password: '12345',
+        engine: 'mysql',
+        host: '127.0.0.1',
+        port: 3306,
```

Comment:
Do we want to have them store in environmental variables and update the README, env file?
The values seem to be shared across multiple files so it is easier if it is written in one place perhaps (localDatabase.ts).